docs/architecture/dapr-for-net-developers/index.md

```diff
@@ -39,29 +39,31 @@ All other marks and logos are property of their respective owners.
 
 Authors:
 
-> **Rob Vettor**, Principal Cloud System Architect/IP Architect - [thinkingincloudnative.com](https://thinkingincloudnative.com/about/), Microsoft
+> **Rob Vettor**, Principal Cloud Solution Architect - [thinkingincloudnative.com](https://thinkingincloudnative.com/about/), Microsoft
 >
-> **Sander Molenkamp**, Principal Cloud Architect/Microsoft MVP - Info Support [])
+> **Sander Molenkamp**, Principal Cloud Architect/Microsoft MVP - [sandermolenkamp.com](https://www.sandermolenkamp.com), [Info Support](https://www.infosupport.com/en/)
 >
-> **Edwin van Wijk**, Principal Software Architect/Microsoft MVP - Info Support [])
+> **Edwin van Wijk**, Principal Solution Architect/Microsoft MVP - [defaultconstructor.com](https://defaultconstructor.com), [Info Support](https://www.infosupport.com/en/)
 
 Participants and Reviewers:
 
 > **Nish Anil**, Senior Program Manager, .NET team, Microsoft
 >
-> **Mark Fussell**, Prinicpal Program Manager, Azure Incubations, Microsoft
+> **Mark Fussell**, Principal Program Manager, Azure Incubations, Microsoft
 >
-> **Yaron Schneider**, Prinicpal Software Engineer, Azure Incubations, Microsoft
+> **Yaron Schneider**, Principal Software Engineer, Azure Incubations, Microsoft
 >
 > **Ori Zohar**, Senior Program Manager, Azure Incubations, Microsoft
 
 Editors:
 
+> **David Pine**, Senior Content Developer, .NET team, Microsoft
+
 > **Maira Wenzel**, Program Manager, .NET team, Microsoft
 
 ## Version
 
-This guide has been written to cover **.NET Core 3.1** version along with many additional updates related to the same “wave” of technologies (that is, Azure and additional third-party technologies) coinciding in time with the .NET Core 3.1 release.
+This guide has been written to cover the **Dapr 1.0** version. .NET Core samples are based on **.NET Core 3.1**.
```

Comment:
Can you fix the link for PDF, please? Replace this `[PDF](https://dotnet.microsoft.com/download/e-book/cloud-native-azure/pdf)` with `https://aka.ms/dapr-ebook`

Reply:
Fixed